pulumi/integration_tests/__main__.py

```diff
@@ -76,7 +76,8 @@ def main() -> None:
         "aws_env_vars_for_local": grapl_stack.aws_env_vars_for_local,
         "aws_region": aws.get_region().name,
         "container_images": _e2e_container_images(artifacts),
-        "stack_name": grapl_stack.upstream_stack_name,
+        # Used by graplctl to determine if it should manual-event or not
```

Comment:
what does "manual-event" mean?

Reply:
![image](https://user-images.githubusercontent.com/69007229/155636613-9a54f0ec-2050-4ffc-832c-40d81ede1822.png)

basically it's the whole "when we put something in S3, send a message to the associated SQS queue"
that's automatic in AWS